src/rgw/rgw_rados.cc

```diff
@@ -2683,7 +2683,7 @@ int RGWPutObjProcessor_Atomic::complete_writing_data()
     obj_len = (uint64_t)first_chunk.length();
   }
   while (pending_data_bl.length()) {
-    void *handle;
+    void *handle = nullptr;
```

Comment:
@agutikov can have this initialization here, but should modify the following `write_data()`  to initialize it by default to avoid any other similar issues later?